overlays/openshift-patches/flags-patch.yaml

```diff
@@ -0,0 +1,9 @@
+---
+- op: add
+  path: /spec/template/spec/containers/0/args/-
+  value:
+    --pipelines-namespace=tekton-pipelines
```

Comment:
Worth adding a comment here I think, so, and correct me if I'm wrong, we do this because

1) In the OpenShift Pipelines operator case, we'll be in `openshift-pipelines` alongside Triggers and Pipelines, and the lookup is done there by default
2) this really solves the non-operator case (as installing from release.notags.yaml for Pipelines or release.yaml for Triggers) as the deployments will end up in tekton-pipelines

Am I right in thinking that? It's a nuance I think we could capture here. Changes otherwise look great and are very much welcome

Reply:
Ok for adding the comment but i have to admit i am a bit blind when it comes to openshift.

From what i understand openshift has two install path: operator and non-operator.

Do the manifest we generate in the releases apply to the operator install path ? or is the dashboard also deployed through the operator and the manifests are not used ?

For the non-operator path i suppose everything is deployed using the release manifests, hence the tekton-pipelines/openshift-pipelines mix.

Reply:
```
Do the manifest we generate in the releases apply to the operator install path ? or is the dashboard also deployed through the operator and the manifests are not used in this case ?
```

They're for the OpenShift Pipelines operator path, preferring the openshift-pipelines namespace - the Dashboard isn't installed as part of that. May be worth adding something into the main README to advise users on that (and mentioning how the namespace can be adjusted accordingly - which this PR demonstrates through the yaml changes).

```
For the non-operator path i suppose everything is deployed using the release manifests, hence the tekton-pipelines/openshift-pipelines mix.
```

yep, exactly

Reply:
So when using OpenShift Pipelines operator, the operator installs pipelines (and triggers ?) in openshift-pipelines namespace and the dashboard is deployed manually using the release manifest ?

Reply:
And OpenShift Pipelines operator is different than tekton operator ?

Reply:
Yeah that's right, in both regards. Haven't used the Tekton operator on OpenShift but consider it beyond the scope of this work (by documenting how to override the lookup we're covered anyway)

Reply:
I ask because i'm trying to implement the logic behind all this in https://github.com/tektoncd/dashboard/pull/1494
Thanks

Reply:
I added a note in docs and comments in the patch file.
Please check it's not complete bullshit ;-)